File: src/string/isGoodPassword/is_good_password.py
def has_digit(password):
    return len([ch for ch in password if ch.isdigit()]) > 0


def has_upper(password):
    return len([ch for ch in password if ch.isupper()]) > 0


def has_lower(password):
    return len([ch for ch in password if ch.islower()]) > 0


def has_big_length(password):
    return len(password) >= 8


def is_good_password(password):
    return has_digit(password) and has_upper(password) and \
        has_lower(password) and has_big_length(password)

File: src/string/isGoodPassword/IsGoodPassword.java
package string.isGoodPassword;

public class IsGoodPassword {
    public static boolean isGoodPassword(String pas) {
        boolean hasDigit = pas.chars()
                .mapToObj(i -> (char) i)
                .anyMatch(c -> Character.isDigit(c));
        boolean hasLower = pas.chars()
                .mapToObj(i -> (char) i)
                .anyMatch(c -> Character.isLowerCase(c));
        boolean hasUpper = pas.chars()
                .mapToObj(i -> (char) i)
                .anyMatch(c -> Character.isUpperCase(c));
        return pas.length() >= 8 && hasDigit && hasLower && hasUpper;
    }

}